docs/concepts/data-views.asciidoc

```diff
@@ -32,51 +30,68 @@ uploaded a file, or added sample data,
 `filebeat-apache-a`, `filebeat-apache-b`, and so on.
 +
 ** To match multiple single sources, enter their names,
 separated by a comma.  Do not include a space after the comma.
-`filebeat-a,filebeat-b` matches two indices, but does not match `filebeat-c`.
+`filebeat-a,filebeat-b` matches two indices.
 +
 ** To exclude a source, use a minus sign (-), for example, `-test3`.
 
-. If {kib} detects an index with a timestamp, expand the *Timestamp field* menu,
+. Open the *Timestamp field* dropdown,
 and then select the default field for filtering your data by time.
 +
-** If your index doesn’t have time-based data, choose *I don’t want to use the time filter*.
-+
 ** If you don’t set a default time field, you can't use
 global time filters on your dashboards. This is useful if
 you have multiple time fields and want to create dashboards that combine visualizations
-based on different timestamps.
+based on different timestamps.z
++
+** If your index doesn’t have time-based data, choose *I don’t want to use the time filter*.
 
-. To display all indices, click *Show advanced settings*, then select *Allow hidden and system indices*.
+. Click *Show advanced settings* to:
+** Display hidden and system indices.
+** Specify your own {data-source} name. For example, enter your {es} index alias name.
 
-. To specify your own {data-source} name, click *Show advanced settings*, then enter the name in the *Custom {data-source} ID* field. For example, enter your {es} index alias name.
+. [[reload-fields]] Click *Save {data-source} to {kib}*.
++
+You can manage your data view from *Stack Management*.
 
-. [[reload-fields]] To use this data in searches and visualizations that you intend to save, click *Save {data-source} to {kib}*.
+[float]
+==== Temporary {data-sources}
 
-. To explore your data without creating a {data-source}, click *Use without saving*.
-+
-This allows you to explore your data in *Discover*, *Lens*, and *Maps*
-without making it visible to others in your space. You can save the {data-source} later
-if you create a search or visualization that you want to share.
+What if you want to explore your data or create a visualization without first saving it as a data view?
+You can do that by selecting *Use without saving* in the *Create {data-source}* form in *Discover*
+or *Lens*. With a temporary {data-source}, you can
+do everything you'd do with a saved
+{data-source},
+including adding fields and creating an {es} query alert. Your work won't be visible to others in your space.
+
+A temporary {data-source} remains in your space until you change apps, or until you save it.
+Temporary {data-sources} are not available in *Stack Management.*
+
+[role="screenshot"]
+image::https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blte3a4f3994c44c0cc/637eb0c95834861044c21a25/ad-hoc-data-view.gif[how to create an ad-hoc data view]
 
 
 [float]
 [[rollup-data-view]]
-==== Create a data view for rolled up data
+==== Data views and rolled up data
```

Comment:
Can this become actionable?

For example, `Use data views with rolled up data`